Examples/Cocoa/Sources/Objective_C/include/Pin.h

```diff
@@ -26,6 +26,7 @@ typedef NS_ENUM(NSInteger, PinAttributionObjectsInternalType) {
 + (instancetype)objectWithUser:(User *)user;
 - (void)matchBoard:(nullable PLANK_NOESCAPE void (^)(Board * board))boardMatchHandler orUser:(nullable PLANK_NOESCAPE void (^)(User * user))userMatchHandler;
 - (BOOL)isEqualToPinAttributionObjects:(PinAttributionObjects *)anObject;
+- (id)dictionaryObjectRepresentation;
```

Comment:
Why is this an `id`?

Reply:
It is this for an ADT since it can return one of many values.

Reply:
Cool sounds good